extensions/sql-migration/src/wizard/skuRecommendation/skuRecommendationPage.ts

```diff
@@ -377,29 +405,53 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		if (!this.migrationStateModel._assessmentResults) {
```

Comment:
Is there a TODO in this empty if block?

Reply:
Yes I am not sure what to show exactly if assessment result is coming null. Maybe we can show all data values as zero. But we can decide later. Added TODO comment for now